### DISCUSSION TITLE
How to connect  FUXA with Node-RED

### DISCUSSION THREAD

**Discussion by Wannapayeot** [frangoteam/FUXA#1262] [Q&A] [answered]
Can you recommend connecting FUXA to Node-RED to read data from a Mitsubishi PLC?

**Comment by unocelli** [comment]
Hi, you can create in Node-RED RestAPI get/post and configure in FUXA a device via environment variable #650 to communicate Tags values

**Reply by Wannapayeot** [reply, marked as answer]
I can't add tags Can you recommend me?

![111](https://github.com/frangoteam/FUXA/assets/22538561/7f6fd670-dad8-4d79-996c-1edf04326f42)

**Reply by unocelli** [reply]
You have to define the WebAPI in Node-RED
